build.sbt

```
@@ -58,145 +54,33 @@ val coreDeps = Seq(
-        "Christopher Davenport",
-        "chris@christopherdavenport.tech",
-        url("https://github.com/ChristopherDavenport")
-      ),
-      Developer(
-        "amesgen",
-        "Alexander Esgen",
-        "amesgen@amesgen.de",
-        url("https://github.com/amesgen")
-      ),
-      Developer(
-        "rossabaker",
-        "Ross A. Baker",
-        "ross@rossabaker.com",
-        url("https://github.com/rossabaker")
-      )
-    ),
-    githubWorkflowJavaVersions := Seq("11", "17").map(JavaSpec.temurin(_)),
-    githubWorkflowBuild := Seq(
-      WorkflowStep
-        .Sbt(List("scalafmtCheckAll", "scalafmtSbtCheck"), name = Some("Check formatting")),
-      WorkflowStep.Sbt(List("headerCheckAll"), name = Some("Check headers")),
-      WorkflowStep.Sbt(List("Test/compile"), name = Some("Compile")),
-      WorkflowStep
-        .Sbt(List("core/mimaReportBinaryIssues"), name = Some("Check binary compatibility")),
-      WorkflowStep.Sbt(
-        List("unusedCompileDependenciesTest", "undeclaredCompileDependenciesTest"),
-        name = Some("Check unused and undeclared compile dependencies")
-      ),
-      WorkflowStep.Sbt(List("test"), name = Some("Run tests")),
-      WorkflowStep.Sbt(List("doc"), name = Some("Build docs"))
-    ),
-    githubWorkflowPublishPostamble := Seq(
-      WorkflowStep.Run(
-        List("""
-             |eval "$(ssh-agent -s)"
-             |echo "$SSH_PRIVATE_KEY" | ssh-add -
-             |git config --global user.name "GitHub Actions CI"
-             |git config --global user.email "ghactions@invalid"
-             |sbt ++${{ matrix.scala }} docs/ghpagesPushSite
-             """.stripMargin),
-        name = Some("Publish docs"),
-        env = Map(
-          "SSH_PRIVATE_KEY" -> "${{ secrets.SSH_PRIVATE_KEY }}",
-          "SBT_GHPAGES_COMMIT_MESSAGE" -> "Updated site: sha=${{ github.sha }} build=${{ github.run_id }}"
-        ),
-        cond = Some("github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v')")
-      )
-    ),
-    githubWorkflowPublishTargetBranches := Seq(
-      RefPredicate.Equals(Ref.Branch("main")),
-      RefPredicate.StartsWith(Ref.Branch("series/")),
-      RefPredicate.StartsWith(Ref.Tag("v"))
-    )
-  )
+val scala213 = "2.13.8"
+ThisBuild / crossScalaVersions := Seq("2.12.15", scala213, "3.1.1")
+ThisBuild / scalaVersion := scala213
+ThisBuild / tlBaseVersion := "1.0"
+ThisBuild / startYear := Some(2019)
+ThisBuild / developers := List(
+  tlGitHubDev("ChristopherDavenport", "Christopher Davenport"),
+  tlGitHubDev("amesgen", "Alexander Esgen"),
+  tlGitHubDev("rossabaker", "Ross A. Baker")
 )
 
-lazy val commonSettings = Seq(
-  testFrameworks += new TestFramework("munit.Framework")
-)
-
-lazy val generateNetlifyToml = taskKey[Unit]("Generate netlify.toml")
+ThisBuild / githubWorkflowJavaVersions := Seq("11", "17").map(JavaSpec.temurin(_))
+ThisBuild / tlCiReleaseBranches := Seq("main")
+ThisBuild / tlSitePublishBranch := Some("main")
```

You're right. Personally I would keep this set to `series/0.7` so that they don't overwrite each other. 0.7 should be fine until we get the versioned site up and running.

```suggestion
ThisBuild / tlSitePublishBranch := Some("series/0.7")
```